RecoTracker/MkFit/plugins/MkFitOutputConverter.cc

```diff
@@ -341,6 +346,10 @@ TrackCandidateCollection MkFitOutputConverter::convertCandidates(const MkFitOutp
     const auto seedIndex = cand.label();
     LogTrace("MkFitOutputConverter") << " from seed " << seedIndex << " seed hits";
 
+    // Rescale candidate error if candidate is already propagated to first layer, to be consistent with TransientInitialStateEstimator::innerState used in CkfTrackCandidateMakerBase
+    // Error is only rescaled for candidates propagated to first layer; otherwise, candidates undergo backwardFit where error is already rescaled
```

Comment:
having some line breaks would be nice.
The comments do not fit in the browser without side-scrolling

Reply:
Done.